pom.xml

```diff
@@ -8,6 +8,38 @@
     <artifactId>Patterns</artifactId>
     <version>1.0-SNAPSHOT</version>
 
+    <dependencies>
```

Comment:
I see both junit and testng in dependencies, remove one of them - they are not used both simultaneously